### ISSUE TITLE
UnusedImport false positive with import kotlinx.android.synthetic as

### ISSUE THREAD

**Issue by olegivo** [shyiko/ktlint#286]
When you use import as for an import alias, ktlint detects a lint error: Unused import
Sample: 
import kotlinx.android.synthetic.main.fragment_searchresultobjects.recyclerview as recyclerView

**Comment by shyiko** [comment]
Can you please provide a complete example?

```sh
$ cat A.kts
import kotlinx.android.synthetic.main.fragment_searchresultobjects.recyclerview as recyclerView
recyclerView()

$ ktlint --debug A.kts
...
[DEBUG] 772ms / 1 file(s) / 0 error(s)

$ ktlint --version
0.28.0
```
Thank you.

**Comment by olegivo** [comment]
I can provide [sanitized project for Android Studio](https://www.dropbox.com/s/45mhjw23deroyz8/ktlint-unused-imports-bug.zip?dl=0) which reproduce the behavior.
Version of ktlint is 28
I use kotliner gradle plugin to run ktlint
In the project there is a gradle task named 'staticAnalysis' that depends on ktlint task.
For now I does not understood yet where the ktlint placed to run it with arguments that you ask.

**Comment by shyiko** [comment]
@olegivo Thank you for that. The line numbers reported by ktlint were off by 1 (CRLF EOL wasn't properly handled). Fixed in [0.29.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0290---2018-10-02).

**Comment by olegivo** [comment]
Thank you very much!
